docs/web-apps/automated-testing/_partials/_advanced-cypress.md

```diff
@@ -37,4 +37,4 @@ saucectl run --env CYPRESS_TAGS="(@smoke or @ui) and (not @slow)"
 
 ## Component Testing
 
-Component Testing is in beta and due to an issue from Cypress, `saucectl` doesn't currently support Component Testing.
+Due to an issue from Cypress, `saucectl` doesn't currently support Component Testing.
```

Comment:
Thank you!

I think we might as well remove the `Due to an issue from Cypress` part. If it's out of beta and working (outside of Sauce), then the current lack of support for Component Testing is simply on us 😅 A good reminder and opportunity to add support for this 👍 

```suggestion
`saucectl` doesn't currently support Component Testing.
```